src/renderer/components/configure-recording-set-page/preview-page.tsx

```diff
@@ -0,0 +1,126 @@
+import log from 'electron-log';
+import React, { FC, useEffect, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import Container from 'react-bootstrap/Container';
+import { CSSTransition } from 'react-transition-group';
+
+import { Consumer } from '../../types';
+import { checkFileExistence, readFile } from '../../utils';
+import ImageButton from '../image-button';
+
+import downButton from './down-button.svg';
+import switchPageButton from './switch-preview.svg';
+
+import { RecordingPageState } from '.';
+
+interface PreviewPageProps {
+  setRecordingSetState: Consumer<RecordingPageState>;
+  pageName: string;
+  transition: boolean;
+  setTransition: Consumer<boolean>;
+  fileName: string;
+  setDropDownState: () => void;
+}
+
+export const PreviewPage: FC<PreviewPageProps> = ({
+  setRecordingSetState,
+  pageName,
+  transition,
+  setTransition,
+  fileName,
+  setDropDownState,
+}) => {
+  const [className, setClassName] = useState<string>('slide-down');
+  const [pageText, setPageText] = useState<string>(pageName + ' not availible');
+
+  const transitionProps = {
+    in: transition,
+    appear: true,
+    timeout: 3000,
+    classNames: className,
+  };
+  useEffect(() => {
```

Comment:
Try to extract these `useEffect` out as custom hooks. By giving them a name, it can make the code more readable.

Reply:
Bump